src/Seq.App.Opsgenie/OpsgenieApp.cs

```diff
@@ -38,6 +42,13 @@ public class OpsgenieApp : SeqApp, IDisposable, ISubscribeToAsync<LogEventData>
             InputType = SettingInputType.Password)]
         public string ApiKey { get; set; }
 
+        [SeqAppSetting(
+            DisplayName = "Suppression interval (seconds)",
+            HelpText =
+                "If an alert has been raised, further alerts will be suppressed for this time. Default 60, Minimum 0.",
+            InputType = SettingInputType.Integer)]
+        public int SuppressionTime { get; set; } = 60;
```

Comment:
Suppression time is provided by Seq (for regular alerts it's called suppression time; for streamed events its "debounce window"), it'd be preferable not to duplicate it here.

Reply:
Removed suppression time